assets/css/src/amp-mobile-version-switcher.css

```
@@ -1,3 +1,9 @@
+/* In the Twenty Twenty-One theme, when on a non-AMP page and the mobile menu is open, the switcher is incorrectly
+shown at the top. In that case the switcher is hidden. */
+body.primary-navigation-open.lock-scrolling > #amp-mobile-version-switcher {
```

The relevant class here is `lock-scrolling`, right? Therefore, maybe the `primary-navigation-open` class can be removed:

```suggestion
body.lock-scrolling > #amp-mobile-version-switcher {
```

Then again, should this CSS not actually go in the core theme sanitizer for Twenty Twenty-One? This style rule isn't going to be relevant to other themes in general, will it?

---

Yea it would make more sense to put it in the core theme sanitizer. Done in https://github.com/ampproject/amp-wp/pull/5648/commits/f9984c3fc2a77e12e53373c8ca95686e3cf2a56a.